source/s3_client.c

```diff
@@ -180,6 +181,16 @@ struct aws_s3_client *aws_s3_client_new(
         return NULL;
     }
 
+    /* Cannot be less than zero.  If zero, use default. */
+    if (client_config->part_size != 0 && client_config->part_size < MB_TO_BYTES(s_s3_min_part_size_MB)) {
+        AWS_LOGF_ERROR(
+            AWS_LS_S3_CLIENT,
+            "Cannot create client from client_config; part size is less than the minimum part size of %d MB.",
```

Comment:
`part size is not zero(default) and less than the minimum part size of %d MB` Or something like that will be better

Reply:
It's subjective, but I think calling out the default case is redundant in the messaging.